hack/build/docker/builder/Dockerfile

```diff
@@ -30,35 +24,37 @@ RUN sed -i 's/proxy = None//gI' /tmp/fedora_ci.dnf.repo && \
 	nbdkit-devel \
 	unzip \
 	java-11-openjdk-devel \
-	btrfs-progs-devel \
-	device-mapper-devel \
-	&& microdnf clean all && \
-	mv /etc/yum.repos.d/old/* /etc/yum.repos.d/ && \
-	rmdir /etc/yum.repos.d/old
+	&& dnf clean all
+
+# Necessary for Bazel to find Python inside the container
+#
+# https://github.com/bazelbuild/bazel/issues/8665
+# https://github.com/bazelbuild/bazel/issues/11554
+RUN ln -s /usr/bin/python3 /usr/bin/python
 
 RUN pip3 install --upgrade j2cli operator-courier==2.1.11 && \
-	curl -sL https://services.gradle.org/distributions/gradle-6.6-bin.zip -o gradle-6.6-bin.zip && \
-	mkdir /opt/gradle && \
-	unzip -d /opt/gradle gradle-6.6-bin.zip && \
-	ln -s /opt/gradle/gradle-6.6/bin/gradle /usr/local/bin/gradle && \
-	rm gradle-6.6-bin.zip
+    curl -sL https://services.gradle.org/distributions/gradle-6.6-bin.zip -o gradle-6.6-bin.zip && \
+    mkdir /opt/gradle && \
+    unzip -d /opt/gradle gradle-6.6-bin.zip && \
+    ln -s /opt/gradle/gradle-6.6/bin/gradle /usr/local/bin/gradle && \
+    rm gradle-6.6-bin.zip
 
 ENV GIMME_GO_VERSION=1.17.5 GOPATH="/go" GO111MODULE="on"
 
 RUN mkdir -p /gimme && curl -sL https://raw.githubusercontent.com/travis-ci/gimme/master/gimme | HOME=/gimme bash >> /etc/profile.d/gimme.sh
 
 RUN \
-	source /etc/profile.d/gimme.sh && \
-	eval $(go env) && \
-	go install github.com/onsi/ginkgo/ginkgo@v1.14.1 && \
-	go install golang.org/x/tools/cmd/goimports@latest && \
-	go install mvdan.cc/sh/cmd/shfmt@latest && \
-	go install github.com/mattn/goveralls@latest && \
-	go install golang.org/x/lint/golint@latest && \
-	go install github.com/rmohr/go-swagger-utils/swagger-doc@latest && \
-	go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.7.0 && \
-	go install github.com/securego/gosec/v2/cmd/gosec@0ce48a5 && \
-	rm -rf "${GOPATH}/pkg"
+    source /etc/profile.d/gimme.sh && \
+    eval $(go env) && \
+    go get github.com/onsi/ginkgo/ginkgo && \
+    go get golang.org/x/tools/cmd/goimports && \
+    go get mvdan.cc/sh/cmd/shfmt && \
+    go get github.com/mattn/goveralls && \
+    go get -u golang.org/x/lint/golint && \
+    go get github.com/rmohr/go-swagger-utils/swagger-doc && \
+    go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.3.0 && \
+    go get github.com/securego/gosec/v2/cmd/gosec@0ce48a5 && \
```

Comment:
Why going back to `go get` Isn't it deprecated?

Reply:
Didn't notice this is a change I should adopt, will change.